### ISSUE TITLE
PLANET-5257 "Teasers" accessibility

### ISSUE THREAD

**Issue by planet-4** [greenpeace/planet4#114]
Teasers are those blocks that show a preview of the post, like the list of posts in the [Articles Block](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/blocks/Articles/ArticlePreview.js) or the different twig templates in master-theme repo starting with `tease-`.

- Make sure the mid dot chars and number sign of the hashtags are not read aloud, this might already be done (check the [Post Type & Hashtags ticket](https://github.com/greenpeace/planet4/issues/86) and the [Articles Accessibility ticket](https://github.com/greenpeace/planet4/issues/83)).
- Then for the rest of the content there, make sure the date, author and other metadata are read as a single piece of text, and with no special characters read aloud. **aria-hidden** and **role=text** will come handy for this.

**Screenshots**

![Screenshot](https://jira.greenpeace.org/secure/attachment/31375/Captura+de+pantalla+2020-06-28+a+la%28s%29+19.00.56.png)
```
Reporter: pcuadrad
Sections: 
```
Potentially affected repositories: [planet4-plugin-gutenberg-blocks](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks), [planet4-master-theme](https://github.com/greenpeace/planet4-master-theme)

### Testing
- You can add the Articles block to a page and test it with a screen reader, the block should also already be present on your local [homepage](http://www.planet4.test/) so you can check that too. You can read the [docs](https://planet4.greenpeace.org/act/articles/) for more details on how the block works.
- For the other teasers, you can check your local [search page](https://www.planet4.test/?s=&orderby=_score)

**Comment by DanceParty** [comment]
Grabbing this one (not able to assign myself)

**Comment by comzeradd** [comment]
Thank you! 💚 Let us know if something is not clear in the ticket description.

**Comment by DanceParty** [comment]
One of the confusing things is around the bullets. The bullets are pseudo elements generated in the css. I don't believe there is a way to hide these type of elements from the screenreader unless I write them in the jsx. I can move forward with that if that is okay. Just wanted to make sure I wasn't missing anything

**Comment by comzeradd** [comment]
You are right, css pseudo elements are hard to hide from screen readers. But indeed, we could just add that bullet as a regular `<span>` in the jsx template so we can use the appropriate markup to "hide it" from screen readers.

**Comment by DanceParty** [comment]
Awesome! Just wanted to make sure that would be the correct approach here. Thanks, hopefully I'll have a PR by end of week :)

**Comment by DanceParty** [comment]
Finally getting around to this, it seems like the "#" is currently read aloud as "hashtag". My understanding of this issue is that we don't want to read "pound sign" or "number sign", but I feel that "hashtag" is appropriate here and I will probably leave it.

**Comment by DanceParty** [comment]
Alright PRs opened, thanks for the patience and would appreciate some feedback whenever you have the chance @comzeradd 🙇

**Comment by IlBrizzo** [comment]
Thank you so much for your help @DanceParty ! We made sure your awesome work is acknowledged [in the release note this ticket was part of](https://planet4.greenpeace.org/story/13023/v2-58/), it's not a bestseller credit, but it's the least we could do! 

Rock on 🤘

**Comment by suzi-greenpeace** [comment]
yay thanks @DanceParty !
